R/linter_tags.R

```diff
@@ -0,0 +1,174 @@
+#' Get Linter metadata from a package
+#'
+#' Obtain a tagged list of all Linters available in a package.
+#'
+#' @param packages A character vector of packages to search for linters.
+#'
+#' @section Package Authors:
+#'
+#' To implement \code{available_linters()} for your package, include a file \code{inst/lintr/linters.csv} in your
+#' package.
+#' The CSV file must contain two columns, 'linter' and 'tags', and be UTF-8 encoded.
```

Comment:
nit: slight tweak here to convey those columns are required, but other columns are allowed. maybe just remove "two"?

Reply:
done